src/rust/src/pkcs7.rs

```diff
@@ -164,6 +166,257 @@ fn encrypt_and_serialize<'p>(
     }
 }
 
+#[pyo3::pyfunction]
+fn decrypt_der<'p>(
+    py: pyo3::Python<'p>,
+    data: CffiBuf<'p>,
+    certificate: pyo3::Bound<'p, x509::certificate::Certificate>,
+    private_key: pyo3::Bound<'p, backend::rsa::RsaPrivateKey>,
+    options: &pyo3::Bound<'p, pyo3::types::PyList>,
+) -> CryptographyResult<pyo3::Bound<'p, pyo3::types::PyBytes>> {
+    decrypt(py, data.as_bytes(), certificate, private_key, options)
```

Comment:
Nit, isntead of having a decrypt function that everything calls, just have the others call `decrypt_der`.

I'm not sure there's a reason for `data` to be `CffiBuf`, you can just make it a `&[u8]` directly.

Reply:
Done! Thought `CffiBuf` was important ^^